views/includes/check-your-answers/beneficial-owner-individual.html

```diff
@@ -85,7 +85,19 @@
     ) ]
   }
 }), rows) %}
-
+{% if hiddenChangeLink == true %}
+{% set rows = (rows.push({
+  key: {
+    text: "Date of birth"
+  },
+  value: {
+    text: boiDateOfBirth
+  },
+  actions: {
+  items: [undefined]
```

Comment:
can we just not have actions or action.items?

Reply:
tested them with Jitin, anything less than this seems to make the bottom border not cover the full width unfortunately